### ISSUE TITLE
[Feature Request] Clear messaging regarding Activity Feed Label hiding.

### ISSUE THREAD

**Issue by dennisrijsdijk** [crowbartools/Firebot#2183]
**Describe the solution you'd like**
Currently, when the activity feed section isn't wide enough (eg. on a less wide screen), labels inside of the events are hidden without clear messaging. I was only able to find out why they were being hidden by looking into the source code. There's currently no messaging about this. While I agree it's better than having the text cut off, it's a confusing scenario to be in when it happens out of the blue.

**Additional context**
I had resized the activity feed slightly to make place for larger chat messages which caused this.

**Comment by MorningStarGG** [comment]
Yeah, this has been a major issue since I started using Firebot about 6 months ago. Would love to see this fixed. :)
